### ISSUE TITLE
output <msName> for fragments

### ISSUE THREAD

**Issue by peterstadler** [Edirom/WeGA-WebApp#276]
Fragments *may* feature a `<msName>` element as 'label'. This should somehow be added to the HTML output in connection with restucturing the apparatus page #258 
NB: the `<msName>` within `msDesc/msIdentifier` must *not* be displayed. This element is just a schema requirement.

e.g. A042807
```xml
<msDesc>
   <msIdentifier>
      <msName>Brief Carl Maria von Webers an Caroline von Weber in Hosterwitz.
         London, Montag, 29. Mai 1826 (Nr. 32)</msName>
   </msIdentifier>
   <physDesc>
      <p>urspr. 1Dbl. (2 b. S.o. Adr.)</p>
      <p>Bl. 2 bis auf 1 cm Rand abgeschnitten<!-- liegt in: Berlin SBB (Weberiana Cl. II A a 3, Nr. 23)--></p>
      <p>Siegelloch</p>
      <p>Echtheitsbestätigung am unteren Rand von Bl. 2r von  Jähns: <hi rend="latintype">Umstehend Adresse des <hi rend="underline" n="1">vor</hi>letzten an seine Gattin nach <hi rend="latintype">Dresden</hi> gerichteten eigen|händigen Briefes von <hi rend="latintype">Carl Maria von Weber</hi>, zum Briefe <hi rend="latintype">No</hi> 32, aus <hi rend="latintype">London</hi> | vom 29. u 30. <hi rend="latintype">Mai</hi> 1826 datirt, gehörig, 5 Tage vor seinem Tode geschrieben. Mit Vor|behalt des Briefes <hi rend="latintype">No</hi> 33 vom 2. <hi rend="latintype">Juni</hi>, das Letzte, was <hi rend="latintype">Weber</hi> in seine Heimath schrieb.</hi></p>
      <p>PSt: [Rundst.:] F 26 / 7 6</p>
   </physDesc>
   <additional>
      <listBibl>
         <bibl n="firstPrint" xml:id="printED"><!--ED: -->MMW II, S. 699–700 (Auszug); Reise-Briefe 1823/1826, S. 213–215</bibl>
         <bibl xml:id="printWeckerlin"><!--/ tV: in frz.: -->Weckerlin, Jean-Baptiste, Musiciana, Paris 1877, S. 191 (Auszug vom 30 Mai; in frz.)</bibl>
         <bibl>Einstein, Alfred: Briefe deutscher Musiker, Zürich 1955, S. 162–164</bibl>
      </listBibl>
   </additional>
   <msFrag xml:id="fragBrief">
      <msIdentifier>
         <country>D</country>
         <settlement key="A130002">Berlin</settlement>
         <repository n="D-B">Staatsbibliothek zu Berlin Preußischer Kulturbesitz, Musikabteilung</repository>
         <idno>WFN - Mus.ep. C.M.v.Weber 238</idno>
         <msName>Brieftext</msName>
      </msIdentifier>
   </msFrag>
   <msFrag xml:id="fragAdresse">
      <msIdentifier>
         <country>D</country>
         <settlement key="A130002">Berlin</settlement>
         <repository n="D-B">Staatsbibliothek zu Berlin Preußischer Kulturbesitz, Musikabteilung</repository>
         <idno>Weberiana Cl. II A a 3, Nr. 23</idno>
         <msName>Umschlagblatt</msName>
      </msIdentifier>
   </msFrag>
</msDesc>
```

**Comment by jakosch** [comment]
I'm not quite sure how the output should look like: Should there be two seperate text sources displayed or one text source with two attached fragments? Currently only `msDesc/msIdentifier` counts as a distinct source. Maybe the output could look something like this? 

![image](https://user-images.githubusercontent.com/4886121/52905650-58757f80-323d-11e9-99cb-cac6cf4378cd.png)

**Comment by peterstadler** [comment]
Looks good 👍 

Thank you!

**Comment by peterstadler** [comment]
Just for the record: `<msName>` within `msDesc/msIdentifier` is to be displayed now and implemented in 031e85b97af3e529ab6c5406eda304b97812a758
